src/analysis/souffle/operations.dl

```diff
@@ -48,11 +48,32 @@ operationToPartialOperandLists([owner, op, result, operandList, attrs], operandL
 operationToPartialOperandLists(operation, tail) :-
   operationToPartialOperandLists(operation, [_, tail]).
 
-// Map full operations to operands.
-.decl operationToOperands(operation:Operation, operand: AccessPath)
+// `listLength` is the length of the `operandList`.
+.decl operandListLength(operandList: OperandList, listLength: number)
 
-operationToOperands(operation, operand) :-
-  isOperation(operation), operationToPartialOperandLists(operation, [operand, _]).
+operandListLength(nil, 0).
+operandListLength([head, tail], tailLen + 1) :-
+  operationToPartialOperandLists(_, [head, tail]), operandListLength(tail, tailLen).
```

Comment:
If you decide to do the renaming in #548, please change this to `operationHasOperandListSuffix`. Similarly, for `operationToPartialAttributeLists`

Reply:
Done.